lib/ansible/modules/cloud/amazon/ec2_lc.py

```diff
@@ -173,7 +173,10 @@ def create_launch_config(connection, module):
     name = module.params.get('name')
     image_id = module.params.get('image_id')
     key_name = module.params.get('key_name')
-    security_groups = module.params['security_groups']
+    try:
+        security_groups = get_ec2_security_group_ids_from_names(module.params.get('security_groups'), ec2_connect(module), vpc_id=None, boto3=False)
```

Comment:
This is a bit magic as in "where did this method come from" but of course, we're still importing `*`. Would be good to get that tidied up.